draft-ietf-httpbis-connect-tcp.md

```diff
@@ -145,11 +145,13 @@ Authentication to a templated TCP proxy normally uses ordinary HTTP authenticati
 
 Clients SHOULD assume that all proxy resources generated by a single template share a protection space (i.e., a realm) ({{?RFC9110, Section 11.5}}).  For many authentication schemes, this will allow the client to avoid waiting for a "401 (Unauthorized)" response before each new connection through the proxy.
 
-## Relationship to the Capsule Protocol
+### Use of the Capsule Protocol
 
-Unlike the datagram-oriented templated HTTP proxying specifications {{CONNECT-UDP}}{{CONNECT-IP}}, this specification does not make use of the Capsule Protocol {{?RFC9297}}.  A future specification could define a procedure for performing TCP proxying using the Capsule Protocol, but no such procedure is defined here.
+In this specification, use of the Capsule Protocol {{?RFC9297}} is OPTIONAL.  Clients MAY request use of the Capsule Protocol by including a "Capsule-Protocol: ?1" header field in the request, and servers MAY confirm its use by echoing this header field in the response.
 
-When implementing this specification, clients and servers MUST NOT send a "Capsule-Protocol: ?1" header field.
+When using the Capsule Protocol, TCP data is sent using a new Capsule Type named DATA ({{data-capsule}}).  The DATA type is only applicable when the use of the Capsule Protocol is optional.  It conveys a payload that should be processed in the same way as ordinary stream contents when the Capsule Protocol is not in use.
```

Comment:
2nd sentence seems to seems broken. Maybe what you meant to state is:
```suggestion
When the use of the Capsule Protocol is negotiated, TCP data is sent using a new Capsule Type named DATA ({{data-capsule}}).  It conveys a payload that should be processed in the same way as ordinary stream contents when the Capsule Protocol is not in use.
```

Reply:
OK, I've reframed this in a way that is hopefully clearer.

Reply:
Thank you for the changes! b4d6675 looks good to me.